rasa/data.py

```diff
@@ -11,6 +11,7 @@
 
 logger = logging.getLogger(__name__)
 MARKDOWN_FILE_EXTENSION = ".md"
+YAML_FILE_EXTENSION = ".yml"
```

Comment:
Do you think we could also support `.yaml` as well? Some users prefer that extension.